### DISCUSSION TITLE
Rocket 0.5 release

### DISCUSSION THREAD

**Discussion by kw217** [rwf2/Rocket#2498] [General]
Hi @SergioBenitez - when is there likely to be another 0.5 RC released, and/or when will 0.5 be fully released? I'm keen to use some of the features currently on the main branch. Thanks.

**Comment by SergioBenitez** [comment]
I'll cut an RC later this evening!

**Reply by rogerta** [reply]
That's great!  Will the new RC will bump sqlx support to 0.6?

**Reply by SergioBenitez** [reply]
Yes. It will include everything in `master`.

**Reply by SergioBenitez** [reply]
I've updated the RC branch for 0.5.0-rc.3. I'll need to do some final testing before releasing on crates.io, but it will be out ASAP.

**Reply by kw217** [reply]
Fantastic - thank you so much! I'm very happy to see this!
